api/src/main/resources/workbench-api.yaml

```diff
@@ -99,6 +101,23 @@ paths:
               schema:
                 $ref: '#/components/schemas/StatusAlert'
       x-codegen-request-body-name: statusAlert
+  /v1/status-alert/{id}:
```

Comment:
Since posting a new banner no longer deletes all the other banners, it seemed necessary to add a delete banner endpoint.